src/main/scala/exercises/07_Palindrome.scala

```diff
@@ -1,13 +1,21 @@
 package exercises
 
 object Palindrome {
-  /** 
-   *  True if the string is a palindrome, after removing all non-alphabetic 
+  /**
+   *  True if the string is a palindrome, after removing all non-alphabetic
    *  characters (e.g., spaces, numbers, and punctuation).
-   *  
+   *
    *  Hint: the Scala collections API is your friend
-   *  http://docs.scala-lang.org/overviews/collections/overview.html 
+   *  http://docs.scala-lang.org/overviews/collections/overview.html
    */
-  def isPalindrome(s: String): Boolean = throw new NotImplementedError
+  def isPalindrome(s: String): Boolean = {
+    //s == s.reverse
+    isPalindromeHelper(s.filter(_.isLetter).toLowerCase())
+  }
 
-}
\ No newline at end of file
+  def isPalindromeHelper(s: String): Boolean = s match {
```

Comment:
Cool that you decided to implement this one using pattern matching rather than using built-in methods.